rpi3.mk

```diff
@@ -192,6 +192,7 @@ endif
 .PHONY: rpi3-firmware-clean
 rpi3-firmware-clean:
 	rm -f $(ROOT)/out/$(RPI3_FIRMWARE_FILE).$(RPI3_FIRMWARE_FILE_EXT)
+	rm -rf $(RPI3_STOCK_FW_PATH)
```

Comment:
Deleting folders from make is a headache. If it's just a single folder, then one can specify the files and just delete the files. But when it's like in the "firmware" here where we have a bunch of folders it's a headache.

If you insist using `rm -rf` putting something more than just a single variable and having the variable **first** in the path and you should be pretty safe. i.e., `rm -rf $(ROOT)/rpi3_firmware`. If you're really paranoid you could even rename the folder (sha1 of the current name) to be absolutely sure that there is no collision and say `rm -rf $(ROOT)/rpi3_firmware_b8eee5117faa675da731b22c7472f3d1a29b14cf` and then it's just as safe as the security of some secure protocols.

Having that said, be careful so it doesn't end up like [this](http://www.independent.co.uk/life-style/gadgets-and-tech/news/man-accidentally-deletes-his-entire-company-with-one-line-of-bad-code-a6984256.html).